<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixViewForm</class>
 <widget class="QWidget" name="MatrixViewForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>375</width>
     <height>244</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolBar" name="mainToolBar">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <addaction name="actionSaveAs"/>
      <addaction name="actionPrint"/>
      <addaction name="separator"/>
      <addaction name="actionHelp"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Символ А:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboA"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="relation">
        <property name="styleSheet">
         <string notr="true">border-style:solid;
border-color:black;
border-width:1px;</string>
        </property>
        <property name="text">
         <string>Отношение</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Символ Б:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboB"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="matrixView"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Конфликты</string>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="listMessages">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/print.png</normaloff>:/icons/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Печать...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/help.png</normaloff>:/icons/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
